ahmad-company-db-model.sql

```diff
@@ -0,0 +1,51 @@
+CREATE TABLE "Company" (
+  "id" INTEGER GENERATED BY DEFAULT AS IDENTITY UNIQUE PRIMARY KEY NOT NULL,
+  "name" VARCHAR UNIQUE,
+  "description" VARCHAR,
+  "website" VARCHAR UNIQUE,
+  "size" VARCHAR,
+  "location" VARCHAR,
+  "industry" VARCHAR,
+  "hidden" BOOLEAN dEFAULT false,
+  "created_at" TIMESTAMP dEFAULT (now()) NOT NULL,
+  "updated_at" TIMESTAMP dEFAULT (now()) NOT NULL
+);
+
+CREATE TABLE "Company_Social_Platform" (
+  "id" INTEGER GENERATED BY DEFAULT AS IDENTITY UNIQUE PRIMARY KEY NOT NULL,
+  "company_id" integer,
+  "url" VARCHAR,
+  "typeId" VARCHAR,
+  "created_at" TIMESTAMP dEFAULT (now()) NOT NULL,
+  "updated_at" TIMESTAMP dEFAULT (now()) NOT NULL
+);
+
+CREATE TABLE "Social_Platform_Type" (
+  "id" INTEGER GENERATED BY DEFAULT AS IDENTITY UNIQUE PRIMARY KEY NOT NULL,
+  "type" VARCHAR UNIQUE,
+  "created_at" TIMESTAMP dEFAULT (now()) NOT NULL,
+  "updated_at" TIMESTAMP dEFAULT (now()) NOT NULL
+);
+
+CREATE TABLE "Company_Feedback" (
+  "id" INTEGER GENERATED BY DEFAULT AS IDENTITY UNIQUE PRIMARY KEY NOT NULL,
+  "companyId" integer,
```

Comment:
Let's keep the naming consistent and have everything in snake case. companyId -> company_id